src/llvm.rs

```diff
@@ -54,16 +61,19 @@ fn find_items(lines: &CachedLines) -> BTreeMap<Item, Range<usize>> {
                     .map(|c| c.as_str())
                     .and_then(|c| Some((c.to_owned(), demangle::demangled(c)?))),
             ) {
-                cur.mangled_name = mangled_name;
-                cur.hashed = format!("{hashed:?}");
+                cur.item.mangled_name = mangled_name;
+                cur.item.hashed = format!("{hashed:?}");
+            }
+        } else if line.starts_with("  ") && !line.starts_with("   ") {
```

Comment:
This feels very fragile. What we we checking here? Is it possible to match for the actual content? Like if it's a comment - strip suffix, strip prefix, check if it is or isn't empty?

Reply:
I agree that this is not really self-explanatory - I am not sure if we should be trying to match on the content though, as that's a bit of a feature creep IMHO; I think a good middle-ground would be to have the filtering logic be a separate function which goes over why this simple check works. 
As far as I can tell, it boils down to:
- The first check filters out comments and labels.
- The second case filters out instruction formatting, e.g:
```
  invoke void @_ZN4bpaf7literal17hd39eb03fefd4e354E(ptr sret(%"bpaf::params::ParseAny<()>") align 8 %_5, ptr align 1 %cmd.0, i64 %cmd.1)
          to label %bb1 unwind label %cleanup, !dbg !4048
```
These two lines are really a single instruction, so by filtering out the lines starting with "   " we make sure such cases are counted properly.
What do you think? I am not willing to die on this hill, but I'd err on the side of a good doc comment than potentially breaking with LLVM update (granted that it is possible with current llvm-lines method as well)

Reply:
Hmm... I see. I guess we can take it. I don't really read a lot of llvm code so this part gets least love and least features like syntax highlighting. Maybe one day :)